x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -2451,6 +2451,8 @@
     "kbnVislibVisTypes.controls.pointSeries.valueAxes.positionLabel": "配置",
     "kbnVislibVisTypes.controls.pointSeries.valueAxes.rotateLabel": "回転",
     "kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBoundsLabel": "データバウンドに合わせる",
+    "kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBounds.bottomMargin": "下余白",
+    "kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBounds.minNeededBottomMargin": "下マージンは0以上でなければなりません",
```

Comment:
It's not necessary to add the translation your self, those new ids are automatically extracted from the code before the next release, checked against existing and translated

Reply:
translation removed